presto-main/src/test/java/com/facebook/presto/cost/TestAggregationStatsRule.java

```diff
@@ -89,10 +87,11 @@ public void testAggregationWhenAllStatisticsAreKnown()
     private StatsCalculatorAssertion testAggregation(VariableStatsEstimate zStats)
     {
         return tester().assertStatsFor(pb -> pb
+                .registerVariable(pb.variable("x"))
                 .aggregation(ab -> ab
-                        .addAggregation(pb.variable("sum", BIGINT), expression("sum(x)"), ImmutableList.of(BIGINT))
-                        .addAggregation(pb.variable("count", BIGINT), expression("count()"), ImmutableList.of())
-                        .addAggregation(pb.variable("count_on_x", BIGINT), expression("count(x)"), ImmutableList.of(BIGINT))
+                        .addAggregation(pb.variable("sum", BIGINT), pb.rowExpression("sum(x)"))
```

Comment:
I wonder if we should create rowExpressions directly rather relying on expression to rowExpression translation. 
If there is a world in which we end up getting rid of expressions, leveraging rowExpressions directly would be useful.

Reply:
That's a different discussion, there's easily 1000+ usages of those in tests. Let's take this discussion offline, outside of this PR.

Reply:
Yeah no need for this PR.